<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Thread1</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <widget class="QPushButton" name="btnCreate1">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>158</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Criar</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="btnSuspend1">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>158</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Suspender</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnActivate1">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>158</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Ativar</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnFinish1">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>110</y>
             <width>158</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Terminar</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Thread2</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <widget class="QPushButton" name="btnCreate2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>158</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Criar</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="btnSuspend2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>158</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Suspender</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnActivate2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>158</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Ativar</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnFinish2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>110</y>
             <width>158</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Terminar</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>605</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
